.github/workflows/action_server_binary_release.yml

```diff
@@ -264,10 +272,21 @@ jobs:
             aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_CICD_DROPBOX }}
             aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_CICD_DROPBOX }}
             aws-region: eu-west-1
-      - name: Copy files to the deploy drop-box with the AWS CLI
+      - name: AWS S3 copies
+        if: ${{ !steps.check_beta.outputs.is_beta }}
         run: |
+          if [ "${{ !steps.check_beta.outputs.is_beta }}" = "true" ]; then
```

Comment:
I think this is redundant, maybe remove line 276 so that this and the else make sense?